crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json

```diff
@@ -11,12 +11,10 @@
 		"priority": -1,
 		"ingredients": [
 			{
-				"item": "minecraft:sandstone",
-				"data": 3
+				"item": "minecraft:red_sandstone"
 			},
 			{
-				"item": "minecraft:sandstone",
-				"data": 3
+				"item": "minecraft:red_sandstone"
```

Comment:
Should be `"minecraft:smooth_sandstone"` not `"minecraft:red_sandstone"`

Reply:
actually 🤓👆
it should be smooth_red_sandstone
we are both wrong XD

Reply:
No? it's smooth sandstone which is used to convert it to smooth red sandstone 😂

Reply:
ah yea its in sandstone dying
which isnt a pack I editted on purpose, minus the change to the version number, a random find + replace must of foobar'd this